osbenchmark/benchmark.py

```diff
@@ -763,6 +776,12 @@ def add_workload_source(subparser):
         help="How many seconds between CPU checks there should be during CPU-based redline testing. (Default: 30)",
         default=ConfigureFeedbackScaling.DEFAULT_CPU_CHECK_INTERVAL
     )
+    test_execution_parser.add_argument(
+        "--visualize",
+        help="Generate HTML visualizations for benchmark results.",
```

Comment:
I'd add a detail on how the visualizations will be stored in test executions directory by default